docs/resources/rds_database.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_database
+
+Manages RDS Mysql database resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+resource "huaweicloud_rds_database" "test" {
+  instance_id   = huaweicloud_rds_instance.test.id
+  name          = var.instance_id
+  character_set = "utf8"
+  description   = "test database"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the RDS database resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the RDS instance id. Changing this will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the database name. The database name contains 1 to 64 characters.
+  The name can only consist of letters, digits, hyphens (-), underscores (_) and dollar signs ($). The total number of
+  hyphens (-) and dollar signs ($) cannot exceed 10. RDS for MySQL 8.0 does not support dollar signs ($). Changing this
+  will create a new resource.
+
+* `character_set` - (Required, String, ForceNew) Specifies the character set used by the database, For example utf8,
+  gbk, ascii, etc. Changing this will create a new resource.
+
+* `description` - (Optional, String) Specifies the database description. The value can contain 0 to 512 characters.
+  This parameter takes effect only for DB instances whose kernel versions are at least 5.6.51.3, 5.7.33.1, or 8.0.21.4.
```

Comment:
if use at least, pls remove 5.7.33.1, or 8.0.21.4

Reply:
mysql 5.6, 5.7 and 8.0 are different and are all evolving